src/dataverk/resources/json_resource.py

```diff
@@ -0,0 +1,51 @@
+from dataverk.utils import file_functions
+
+from dataverk.resources.base_resource import BaseResource
+
+
```

Comment:
Har vi fått JSON ressurs? :O

Reply:
Ja, i stedet for å embedde visualiseringer i datapackage.json så vil disse nå skrives til bucket storage som egne json filer. Frontend trenger å laste inn hele datapackage.json fra s3 for å vise noe som helst og når det er flere, store visualiseringer i en datapakke tar dette lang tid. Med denne endringen vil i stedet kun en lenke til plotly og altair visualiseringene ligge i datapackage.json.

Reply:
Ja, det virker fornuftig!